app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence (AI).
```

Comment:
@caitlin-reidy can we reword this bit  **rewarding careers** as it's repeating what's in the 1st sentence. not sure if it needs a comma after careers? **from**... doesn't read quite right to me.